docs/how-to-release.md

```diff
@@ -1,16 +1,24 @@
 
 ## How to Release
 * Prepare release PR:
-    * Pull the changes in the helm-charts sub module folder to get the latest main.
-      * `cd helm-charts && git pull origin main`.
+    * Pull the changes in the helm-charts submodule folder to get the latest main.
+      * `cd helm-charts`
+      * `git submodule update --init` - if submodule was not initialised before
+      * `git pull origin main`
     * Update any changing versions in [release.json](../release.json).
+      * `operator` - always when doing a release 
+      * `version-upgrade-hook`, `readiness-probe` - whenever we make changes in the [versionhook](../cmd/versionhook) or [readiness](../cmd/readiness) files
+      * `agent` - bump to the newest version available
+      * `agent-tools-version` - bump to the newest version available :warning: how to get the newest version?
```

Comment:
> bump to the newest version available :warning: how to get the newest version?
@mircea-cosbuc could help answer this one?

Reply:
The latest `agent-tools-version` is coming from the release branch of the agent we're publishing.